packages/api/src/command/medical/patient/handle-data-contributions.ts

```diff
@@ -51,26 +52,38 @@ export async function handleDataContribution({
     fhirBundle: validatedBundle,
     destinationKey: fhirBundleDestinationKey,
   });
-  const patientDataPromise = async () => {
-    return createOrUpdateConsolidatedPatientData({
-      cxId,
-      patientId: patient.id,
-      fhirBundle: validatedBundle,
-    });
-  };
+  const consolidatedDataUploadResults = await createOrUpdateConsolidatedPatientData({
```

Comment:
Why moved away from a fn that returned a promise? I don't see `consolidatedDataUploadResults` being used until the `Promise.all`.

Reply:
Because now the new data needs to end up in the FHIR server, so it can later be fetched for CCD generation